apps/yara/misc_types.h

```diff
@@ -172,6 +172,8 @@ struct YaraFMConfig
 
     // Sparse SA sampling rate.
     static const unsigned SAMPLING =                    10;
+    static const unsigned WORDS_PER_BLOCK =             0;
+    static const unsigned LEVELS =                      1;
```

Comment:
Just curious, do you know if 2-Levels FM-index is equal/faster/slower than 1-Level?